### PR TITLE
DT-664, DT-1358 and DT-1249: Add ICS2 to eBL and modify API title and remove party prefix from references

### PR DESCRIPTION
**Pull request by HenrikHL**
[DT-664](https://dcsa.atlassian.net/browse/DT-664) - Adds `ICS2` to eBL (also affects BKG)
[DT-1358](https://dcsa.atlassian.net/browse/DT-1358) - Modify eBL API title
[DT-1249](https://dcsa.atlassian.net/browse/DT-1249) - Remove party-prefix from references
[DT-1510](https://dcsa.atlassian.net/browse/DT-1510) - Include displayedAddress in `NotifyParty`

[DT-664]: https://dcsa.atlassian.net/browse/DT-664?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[DT-1358]: https://dcsa.atlassian.net/browse/DT-1358?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[DT-1249]: https://dcsa.atlassian.net/browse/DT-1249?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[DT-1510]: https://dcsa.atlassian.net/browse/DT-1510?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: pint/v3/EBL_PINT_v3.0.0.yaml [hunk 1]
```diff
@@ -3340,19 +3396,19 @@ components:
...
             A reference linked to the `Consignee`.
           example: HHL007
-        consigneesPurchaseOrderReference:
+        purchaseOrderReference:
           type: string
           pattern: ^\S(?:.*\S)?$
           maxLength: 35
           description: |
-            A reference linked to the `Consignee Purchase Order`.
+            A rurchase order eference linked to the `Consignee`.
```
Comment (palatsangeetha) [inline pint/v3/EBL_PINT_v3.0.0.yaml:3411]: Type in purchase and reference

File: ebl/v3/issuance/EBL_ISS_v3.0.0.yaml [hunk 2]
```diff
@@ -2865,6 +2922,70 @@ components:
...
+          example: IKEA Denmark
+        typeOfPerson:
+          type: string
+          maxLength: 50
+          pattern: ^\S(?:.*\S)?$
+          description: |
+            Can be one of the following values as per the Union Customs Code art. 5(4):
+            - `NATURAL PERSON` (A person that is an individual living human being)
+            - `LEGAL PERSON` (person (including a human being and public or private organizations) that can perform legal actions, such as own a property, sue and be sued)
+            - `ASSOCIATION OF PERSONS` (Not a legal person, but recognised under Union or national law as having the capacity to perform legal acts)
```
Comment (palatsangeetha) [inline ebl/v3/issuance/EBL_ISS_v3.0.0.yaml:2946]: 'U' in Union

File: ebl/v3/EBL_v3.0.0.yaml [hunk 3]
```diff
@@ -6114,9 +6385,33 @@ components:
...
-            - `NI` (Other Notify Party)
+            - `CSR` (Consolidator)
+            - `MFR` (Manufacturer)
+            - `WHK` (Warehouse Keeper)
+          example: DDS
+      required:
+        - party
+        - partyFunction
+
+    OtherDocumentPartyHBoL:
```
Comment (palatsangeetha) [inline ebl/v3/EBL_v3.0.0.yaml:6396]: is it HBL?

Comment (palatsangeetha) [inline ebl/v3/EBL_v3.0.0.yaml:6396]: Do we need it as HBoL? as in the isHBLIssued it is just HBL, don't we have to make House Bill of Lading in attribute name consistent?

File: ebl/v3/EBL_v3.0.0.yaml [hunk 4]
```diff
@@ -6232,23 +6542,67 @@ components:
...
           pattern: ^\S(?:.*\S)?$
           maxLength: 35
           description: |
-            A reference linked to the `Shipper Purchase Order`.
+            A purchase order reference linked to the `Shipper`.
           example: HHL007
       required:
         - partyName
 
+    ShipperHBoL:
```
Comment (palatsangeetha) [inline ebl/v3/EBL_v3.0.0.yaml:6562]: Same as above regarding HBL

File: ebl/v3/EBL_v3.0.0.yaml [hunk 5]
```diff
@@ -8096,37 +8705,187 @@ components:
...
+          $ref: '#/components/schemas/Facility'
+        UNLocationCode:
+          type: string
+          pattern: ^[A-Z]{2}[A-Z2-9]{3}$
+          minLength: 5
+          maxLength: 5
+          description: |
+            The UN Location code specifying where the place is located. The pattern used must be
+
+            - 2 characters for the country code using [ISO 3166-1 alpha-2](https://www.iso.org/obp/ui/#iso:pub:PUB500001:en)
```
Comment (palatsangeetha) [inline ebl/v3/EBL_v3.0.0.yaml:8842]: don't we need to remove the mentioning of alpha-2 here?

File: ebl/v3/issuance/EBL_ISS_v3.0.0.yaml (hunk 2 above)
Comment (HenrikHL) [reply ebl/v3/issuance/EBL_ISS_v3.0.0.yaml]: `N` has been capitalized instead

File: ebl/v3/EBL_v3.0.0.yaml [hunk 6]
```diff
@@ -8096,37 +8705,187 @@ components:
...
+
+    PlaceOfAcceptance:
+      type: object
+      title: Place of Acceptance
+      description: |
+        An object to capture `Place of Acceptance` location specified as: identification of the seaport, freight terminal or other place at which the goods are taken over from the shipper by the Ocean Transport Intermediary (OTI) issuing the `House Bill of Lading`.
+
+        **Condition:** Mandatory if different from `Place of Receipt` at the `Master Transport Document` level.
+
+        The location can be specified in **any** of the following ways: `UN Location Code`, `Facility` an `Address` or a `GeoCoordinate`.
```
Comment (palatsangeetha) [inline ebl/v3/EBL_v3.0.0.yaml:8819]: a missing ',' after Facility

**Pull request merged**